pkg/apis/pipeline/v1alpha1/container_replacements.go

```diff
@@ -0,0 +1,72 @@
+/*
+ Copyright 2019 The Tekton Authors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/tektoncd/pipeline/pkg/substitution"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func ApplyContainerReplacements(step *corev1.Container, stringReplacements map[string]string, arrayReplacements map[string][]string) {
```

Comment:
Looks like this doesn't need to be exported, except to be tested in container_replacements_test.go.

Can we either test this as an unexported method, or only test the exported surface in apply_test.go?

Reply:
@ImJasonH it's also used in https://github.com/tektoncd/pipeline/pull/1770/files#diff-768a3668fda546db3140aa35963e1247R25, not sure if that still warrants not exporting (I'm super new to Go, so the import/export semantics elude me a bit right now)

Reply:
Indeed, looks like this is used by the resources package as well and so needs to live on as exported for the time being 👍